### PR TITLE
Hide print opation in stock UOM field

### PR DESCRIPTION
**Pull request by PITPL-Gayatri-Gajjjar**

### CODE REVIEW

**Review by PITPL-Akshay-Prajapati — CHANGES_REQUESTED** [review]
tab spacing issues will create Conflict. make sure to push changes only related to your field not tabbing. A standard spacing(tabing space) should be 4.

File: erpnext/buying/doctype/purchase_order_item/purchase_order_item.json [hunk 1]
```diff
@@ -1,935 +1,934 @@
 {
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/buying/doctype/purchase_order_item/purchase_order_item.json]: don't push indention change.

File: erpnext/stock/doctype/purchase_receipt_item/purchase_receipt_item.json [hunk 2]
```diff
@@ -1,1064 +1,1062 @@
...
-    "module": "Stock",
-    "name": "Purchase Receipt Item",
-    "owner": "Administrator",
-    "permissions": [],
-    "quick_entry": 1,
-    "sort_field": "modified",
-    "sort_order": "DESC"
-   }
\ No newline at end of file
+ "actions": [],
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/stock/doctype/purchase_receipt_item/purchase_receipt_item.json]: don't push indention change.

**Review by PITPL-Akshay-Prajapati — CHANGES_REQUESTED** [review]

File: erpnext/buying/doctype/purchase_order_item/purchase_order_item.json (hunk 1 above)
Comment (PITPL-Gayatri-Gajjjar) [reply erpnext/buying/doctype/purchase_order_item/purchase_order_item.json]: Done

File: erpnext/stock/doctype/purchase_receipt_item/purchase_receipt_item.json (hunk 2 above)
Comment (PITPL-Gayatri-Gajjjar) [reply erpnext/stock/doctype/purchase_receipt_item/purchase_receipt_item.json]: Done

**Pull request closed without merging**